CMPUT301F24bubble/bubbletracks
# update poster, create facility, manage facility, some bug fixes

Testing
- create facility, check if the facility is created in database, the organizer should also have the facility's id in the database
- manage facility, check if the create facility button has been changed  to manage facility. create events using facility. 
- check input validation for event creation. dates in the past not allowed, registration open < registration close < event date
- there is a loading screen between create event and the qr code
- event should be updated to have the facility and and facility should have event id as well
- check update poster functionality for hosted events. see if the poster is updated in the database
- when scanning a qr code to view an event that does not exist it goes to event does not exist screen. there is also a loading screen instead of the default event screen.
